cscs-checks/prgenv/environ_check.py

```diff
@@ -62,9 +62,9 @@ def __init__(self, module_name):
             sn.assert_found(f'{envvar_prefix}_VERSION', self.stderr)
         ])
 
-        # These modules should be fixed in later releases
+        # FIXME: These modules should be fixed in later releases
         cdt = osext.cray_cdt_version()
-        if (cdt and cdt <= '20.10' and
+        if (cdt and cdt <= '20.11' and
             module_name in ['cray-petsc-complex',
                             'cray-petsc-complex-64',
                             'cudatoolkit', 'gcc']):
```

Comment:
`gcc` doesn't fail anymore, in 20.11, so maybe it should be in a different if statement